invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/subjects.html

```diff
@@ -9,30 +9,23 @@
 
 {%- set subjects = record.metadata.subjects -%}
 
-{%- if subjects -%}
-  {%- for scheme, subjects in subjects|selectattr("scheme")|groupby('scheme')|sort(attribute='scheme') %}
-  <div class="ui">
-    <div class="subject-scheme column">
-      <h2 class="ui subject header">{{scheme}}</h2>
-    </div>
-    <div class="subject-label column">
-      {%- for subject in subjects %}
-          <span class="subject">{{ subject.subject}}</span>
-        {%- endfor %}
-    </div>
-  </div>
-  {%- endfor %}
+{% if subjects %}
   {% set keywords = subjects|rejectattr("scheme")|list %}
   {% if keywords %}
-  <div class="ui">
-    <div class="subject-scheme column">
-      <h2 class="ui subject header">{{_('Keywords')}}</h2>
-    </div>
-    <div class="subject-label column">
+  <dl class="details-list" aria-label="keywords">
+    <dt class="hidden">Keywords</dt>
     {%- for subject in keywords %}
-      <span class="subject">{{ subject.subject}}</span>
+      <dd class="subject">{{ subject.subject}}</dd>
+    {%- endfor %}
+  </dl>
+  {% endif %}
+
+  {%- for scheme, subjects in subjects|selectattr("scheme")|groupby('scheme')|sort(attribute='scheme') %}
+  <dl class="details-list" aria-label="subjects">
+    <dt class="ui tiny header">{{scheme}}</dt>
+    {%- for subject in subjects %}
+      <dd class="subject">{{ subject.subject}}</dd>
```

Comment:
Subjects should be linked to a search result page over the subjects. Something along the lines of.

``q=metadata.subjects.subject:<subject>``